indigo_app/views/works.py

```diff
@@ -1177,11 +1188,12 @@ def post(self, request, *args, **kwargs):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        form = self.Form(self.country, self.locality, self.request.POST)
-        form.full_clean()
 
-        context["actual_duplicate"] = form.find_actual_duplicate(self.request.GET.get('pk'))
-        context["possible_duplicates"] = form.find_possible_duplicates(self.request.GET.get('pk'))
+        if self.request.GET.get('pk'):
```

Comment:
this looks like duplicates will only be looked for if the work already exists? which isn't what we want, if I'm reading this right — the duplicate finder should help people to realise that they're about to create a possible duplicate work as well.

Reply:
reverted this, it was breaking when the pk was null, but that seems to be fixed